docs/api.md

```diff
@@ -291,24 +291,40 @@ articles.sync({
   });
 ```
 
-The synchronization updates the local data, and provides information about performed operations.
-Sample result:
+## The synchronization result object
+
+When the `#sync()` promise is fulfilled, a result object is returned, providing information about the performed operations.
+
+Here's a sample result object:
 
 ```js
 {
   ok: true,
   lastModified: 1434270764485,
-  conflicts: [], // Outgoing and incoming conflicts
-  errors:    [], // Errors encountered, if any
-  created:   [], // Created locally
-  updated:   [], // Updated locally
-  deleted:   [], // Deleted locally
-  skipped:   [], // Skipped imports
-  published: [], // Successfully published
-  resolved:  [], // Resolved conflicts, according to selected strategy
+  conflicts: [],
+  errors:    [],
+  created:   [],
+  updated:   [],
+  deleted:   [],
+  skipped:   [],
+  published: [],
+  resolved:  [],
 }
 ```
 
+The synchronization result object exposes the following properties:
+
+- `ok`: The boolean status of the synchronization operation; `true` if no unsolvable conflicts and no errors were encountered.
+- `lastModified`: The timestamp of the latest revision of server data.
```

Comment:
> Is that one timestamp for the whole collection?

Yes

> This timestamp is in milliseconds since 1970, right?

Yes

> Does the server guarantee that any changes that happen immediately after one of the clients synced happen in the next millisecond?

Yes.